#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_fix_manpagesection.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix NAME macro 

@DPATCH@
diff -urNad dansguardian-2.9.9.3~/doc/dansguardian.8 dansguardian-2.9.9.3/doc/dansguardian.8
--- dansguardian-2.9.9.3~/doc/dansguardian.8	2008-03-24 15:33:05.000000000 +0100
+++ dansguardian-2.9.9.3/doc/dansguardian.8	2008-03-24 15:33:51.000000000 +0100
@@ -1,7 +1,8 @@
-.\" -*- nroff -*-
-.TH dansguardian 8
+.TH DANSGUARDIAN "8" Oct 2006" "dansguardian"
+
 .SH NAME
-DansGuardian 2
+dansGuardian2 \- web content filter 
+
 .SH SYNOPSIS
 /etc/rc.d/init.d/dansguardian [stop|start|status|restart]
 .SH DESCRIPTION
